shared/cfg/guest-os/Linux/Fedora/23.ppc64le.cfg

```diff
@@ -0,0 +1,19 @@
+- 23.ppc64le:
+    image_name = images/f23-ppc64le
+    vm_arch_name = ppc64le
+    os_variant = fedora23
+    no unattended_install..floppy_ks
+    boot_path = ppc/ppc64
+    unattended_file_kernel_param_name = ks
+    unattended_install, svirt_install:
+        kernel_params = 'repo=cdrom:/dev/disk/by-label/Fedora-Server-23-ppc64le'
```

Comment:
Well the problem of this setting is, that it only works with `in_cdrom_ks` variants, it was the same story with `RHEL` image, which used to use `sr1` of `sr0` according to test type. Now it's using only `cdrom`, which works everywhere (except RHEL.6 on x86_64).